xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/viewers/attachments.js

```diff
@@ -249,11 +280,12 @@ require(['jquery', 'xwiki-events-bridge'], function($) {
       label = label.replace("__number__", attachmentsNumber);
       $('#tmAttachments').contents().last()[0].nodeValue=label;
     }
-  };
+  }
```

Comment:
Can this `;` really be removed? It seems so, but it also seems inconsistent to me.